docs/reference/modules/snapshots.asciidoc

```diff
@@ -228,9 +228,7 @@ When you restore a source only snapshot:
 
  * The restored index is read-only and can only serve `match_all` search or scroll requests to enable reindexing.
 
- * Queries other than `match_all` will return no results.
-
- * `_get` requests are not supported.
+ * Queries other than `match_all` and `_get` requests are not supported.
```

Comment:
this reads in two ways - you can also read this as if `_get` works (if you don't understand that `_get` is not what we see as a query)